uploader/src/main/resources/cfn-template.yaml

```diff
@@ -0,0 +1,139 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "A game-changing pipeline for uploading video to the internet"
+
+Parameters:
+  BuildBucket:
+    Description: "The S3 bucket where builds live"
+    Type: "String"
+  ConfigBucket:
+    Description: "The S3 bucket where configuration lives"
+    Type: "String"
+  App:
+    Description: "App name"
+    Type: "String"
+    Default: "media-atom-pipeline"
+  Stack:
+    Description: "Stack name"
+    Type: "String"
+    Default: "media-service"
+  Stage:
+    Description: "Stage name"
+    Type: "String"
+    Default: "DEV"
+  UploadBucketName:
+    Description: "The S3 bucket where videos are uploaded to"
+    Type: "String"
+  UploadProgressTable:
+    Description: "The Dynamo table to store progress"
+    Type: "String"
+
+Conditions:
+  CreateProdResources: !Equals [!Ref "Stage", "PROD"]
+  CreateCodeResources: !Equals [!Ref "Stage", "CODE"]
+  CreateDevResources:  !Equals [!Ref "Stage", "DEV"]
```

Comment:
All credit to membership for working out how to do this: https://github.com/guardian/support-workers/blob/master/cloud-formation/src/cfn-template.yaml